### ISSUE TITLE
dist not working in electron to view index.html

### ISSUE THREAD

**Issue by biancode** [fh1ch/node-bacstack-browser#3]
maybe dist is to build on start?

  `"start": "ng build --prod && electron . | ng serve"`

**Comment by fh1ch** [comment]
@biancode yep, your right. I haven't invested too much time in the development setup and couldn't find any Electron / Angular 2 boilerplate which satisfied me. The dist build is only practical if one plans to do a release. I personally uncomment following line (https://github.com/fh1ch/node-bacstack-browser/blob/master/main.js#L13) while actively developing.

I will improve the developer workflow and also come up with a way to easily build a production release.

**Comment by dopry** [comment]
@fh1ch, It would be nice if LOAD_URL came from an environment variable at run time. I typically use the webpack EnvironmentPlugin for this functionality in my build pipelines. 

Angular CLI doesn't support the environment plugin, but they may have an alternative environment handling option.
